libs/npress/AdminModule/presenters/PagesPresenter.php

```diff
@@ -198,6 +209,14 @@ public function pageEditFormSubmitted(Form $form)
     $this->invalidateControl('editform_seoname');
 
     //save values
+    PagesModel::addPagesHistory(
+      $values['id_page'],
+      $this->lang,
+      $values['name'],
+      $values['seoname'],
+      $values['heading'],
+      $values['text']
```

Comment:
Asi by bylo čitelnější zduplikovat to jak se to dělá při `$this->page->save($values)`.

Čili `$this->page->addHistoryVersion($values)`